framework/src/meshgenerators/AnnularMeshGenerator.C

```diff
@@ -160,6 +171,8 @@ AnnularMeshGenerator::generate()
   }
 
   // add nodes at smaller radii, and connect them with elements
+  Real outer_r = _rmax;
+  const Real ring_area = (_rmax * _rmax - _rmin * _rmin) / _nr;
```

Comment:
I would move this in the scope with `if (_equal_area)`. Sure, it's a little more expensive (the compiler may even optimize it out), but it makes it a little more obvious to the fact that it's only used with `_equal_area == true`